tests/DynamoDbCompositeModelTest.php

```diff
@@ -80,6 +80,79 @@ public function testFindRecord()
         $this->assertEquals($seedName, $item->name);
     }
 
+    public function testFirstOrNew()
+    {
+        DynamoDbModel::unguard();
+        $seed = $this->seed();
+        $seedId = array_get($seed, 'id.S');
+        $seedId2 = array_get($seed, 'id2.S');
+        $seedName = array_get($seed, 'name.S');
+
+        $item = $this->testModel->firstOrNew(['id' => $seedId, 'id2' => $seedId2], ['name' => str_random()]);
+        $this->assertNotEmpty($item);
+        $this->assertEquals($seedId, $item->id);
+        $this->assertEquals($seedId2, $item->id2);
```

Comment:
Could make this shorter with something like: `$this->assertEquals([$seedId, $seedId2], [$item->id, $item->id2])` 

Same idea for below assertions.